packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -1559,7 +1559,9 @@ async function handler(
       if (!Number.isNaN(paymentAppData.price) && paymentAppData.price > 0 && !!booking) {
         const credentialPaymentAppCategories = await prisma.credential.findMany({
           where: {
-            userId: organizerUser.id,
+            ...(paymentAppData.credentialId
+              ? { id: paymentAppData.credentialId }
+              : { userId: organizerUser.id }),
```

Comment:
nit: I think we could just use `credentialId` in both team App and non team app installation case. It would simplify the code.

Reply:
This was for backwards compatibility if current `paymentAppData` doesn't contain the `credentialId`

Reply:
Ah makes sense.